packages/commonwealth/client/styles/components/sidebar/index.scss

```diff
@@ -60,7 +60,7 @@
     position: absolute;
     z-index: 70;
 
-    animation: slidein 0.2s ease-in-out;
+    // animation: slidein 0.2s ease-in-out;
```

Comment:
1. please do not commit commented out code
2. why exactly it is commented out?

Reply:
The issue is that whenever we navigate to the notification or explore community section, the sliding sidebar gives the impression of closing and reopening.

Reply:
Just tested and the sidebar remains open after navigations, but I suggest to not remove the animation. Thanks!

Reply:
@mzparacha  if we don't remove the animation we need to change the overall logic of the side bar rendering which will be complicated . i think currently it give the same user experience as old .  looking forward for your feedback

https://github.com/user-attachments/assets/d5fa5e21-d7e8-43a0-b913-310cf80bd038